proto/interchain_security/ccv/provider/v1/keyassignment.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package interchain_security.ccv.provider.v1;
+
+option go_package = "github.com/cosmos/interchain-security/x/ccv/provider/types";
+
+import "gogoproto/gogo.proto";
+import "tendermint/crypto/keys.proto";
+
+message LastUpdateMemo {
+  tendermint.crypto.PublicKey consumer_key = 1;
+  tendermint.crypto.PublicKey provider_key = 2;
+  uint64 vscid = 4;
+  int64 power = 5;
```

Comment:
I forgot- why is the power on here?

Reply:
Added a comment. It's necessary to compute which updates to send and which updates to prune.